ethsigner/signer/multiplatform/src/main/java/tech/pegasys/ethsigner/signer/multiplatform/SignerType.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.signer.multiplatform;
+
+public enum SignerType {
+  FILE_BASED_SIGNER("file-based-signer");
+  private final String type;
+
+  SignerType(String type) {
+    this.type = type;
+  }
+
+  public String getType() {
```

Comment:
rather than exposing the underlying string, it might be better to have a "fromString" - but you'll also need an equals/hashcode - coz then we can write:
if(SignerType.fromString(toml.getString("SignerType")).equals(SIgnerType.FILE_BASED_SIGNER))

And you don't need to leak the string representation of the file based signer enum - its same-same ... but think I'd prefer that...